src/app/modules/pregame/level-selection/level-selection.component.ts

```diff
@@ -168,10 +165,21 @@ export class LevelSelectionComponent {
     console.log(this.maxQuestions);
 
     if (this.maxQuestions === 0) {
-      alert('Please select a level');
+      this.openMissingLevelSelectionModal();
     } else {
       this.cg.createGame(this.maxQuestions, this.playerNames);
       this.router.navigate(['game']);
     }
   }
+
+  openMissingLevelSelectionModal(): void {
+    this.handleMissingLevelSelectionModalResult(this.modalService.open(MissingLevelSelectionModalComponent, { backdrop: 'static' }).result);
+  }
+
+  handleMissingLevelSelectionModalResult(p: Promise<unknown>) {
+    p.then(
+      () => {},
+      () => {}
+    );
+  }
```

Comment:
If you do nothing with the modal, no need for a separate function to handle the result. You need, however, to handle a possible promise rejection as it would add an error message in the console. This can be done with a 'simple' `[...].result.catch(() => {})`

That said, its far from perfect and wee see the limitation of having dismiss reject the promise as we can't 'encode' do nothing behavior by just ignoring the promise because it can be rejected.